src/Arcus.Messaging.Abstractions.ServiceBus/MessageHandling/AzureServiceBusMessageHandler.cs

```diff
@@ -9,6 +9,7 @@ namespace Arcus.Messaging.Abstractions.ServiceBus.MessageHandling
     /// <summary>
     /// Represents a <see cref="IAzureServiceBusMessageHandler{TMessage}"/> template to control the Azure Service Bus message operations during the handling of the deserialized message.
     /// </summary>
+    [Obsolete("Will be removed in v3.0, please use the Azure service bus operations on the " + nameof(AzureServiceBusMessageContext) + " instead")]
```

Comment:
Shouldn't the deprecation description indicate that the `IAzureServiceBusMessageHandler` interface must be used instead ?

Reply:
Ah yes, we can add that as well. Thx! 👍

Reply:
Done that here: https://github.com/arcus-azure/arcus.messaging/pull/551/commits/ae51309029e63e8b3e78c93d1e7c0a9463741b03